x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/rule/RuleExecutor.java

```diff
@@ -152,6 +152,9 @@ protected ExecutionInfo executeWithInfo(TreeType plan) {
                 batchRuns++;
 
                 for (Rule<?, TreeType> rule : batch.rules) {
+                    if (log.isTraceEnabled()) {
+                        log.trace("Trying to apply rule {}", rule);
```

Comment:
"About to apply rule" - trying implies that it might not be applied.

Reply:
Which can be the case... But I prefer the "About to apply rule"